src/pg/sql/45_observatory_mvt.sql

```diff
@@ -383,38 +377,13 @@ DECLARE
   simplification_tolerance NUMERIC DEFAULT 0;
   area_normalization TEXT DEFAULT '';
   i INTEGER DEFAULT 0;
-  clipped TEXT default '';
 BEGIN
+  mc_schema = country || mc_schema;
+
   IF area_normalized THEN
     area_normalization := '/area_ratio';
   END IF;
 
-  IF shoreline_clipped THEN
-    clipped := '_clipped';
-  END IF;
-
-  CASE
```

Comment:
The removal of this is not evident to me, could you explain it a little bit? It looks like `simplification_tolerance` should be handled by the user. What's the purpose of removing the `case`? May this logic be interesting either in a function or in a comment?